classes/PodsInit.php

```diff
@@ -500,14 +498,13 @@ public function register_assets() {
 				'jquery',
 				'pods-dfv',
 				'pods-i18n',
-				'pods-json',
 				'jquery-qtip2',
 			),
 			PODS_VERSION,
 			true
 		);
 
-		wp_register_script( 'pods-cleditor', PODS_URL . 'ui/js/jquery.cleditor.min.js', array( 'jquery' ), '1.3.0' );
+		wp_register_script( 'pods-cleditor', PODS_URL . "ui/js/cleditor/jquery.cleditor{$suffix_min}.js", array( 'jquery' ), '1.4.5', true );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 73334969fd30e458d79abe2476fdb1f2</sub>